PubnativeLite/PubnativeLite/Ad Request/HyBidAdRequest.m

```diff
@@ -143,7 +103,7 @@ - (void)setIntegrationType:(IntegrationType)integrationType withZoneID:(NSString
 
 - (void)setVideoIntegrationType:(IntegrationType)integrationType withZoneID:(NSString *)zoneID {
     if (!zoneID || zoneID.length == 0) {
-        self.zoneID = [self determineZoneIDForVASTForAdSize:self.adSize];
+        [self.delegate request:self didFailWithError: [HyBidError errorWithCode:REQUEST_ALREADY_RUNNING]];
```

Comment:
This has a different Error code than `setIntegrationType:`

Reply:
Thanks for the catch!